MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="SmartDeviceLink RAPI" version="7.0.0" minVersion="1.0" date="2020-11-04">
+<interface name="SmartDeviceLink RAPI" version="7.1.0" minVersion="1.0" date="2020-11-04">
```

Comment:
Merge in the latest develop branch to remove the need for this change

Reply:
Done